lib/tools/adb-commands.js

```diff
@@ -400,19 +404,31 @@ methods.toggleGPSLocationProvider = async function toggleGPSLocationProvider (en
  *        or to a restricted greylist for your target API level.
  */
 methods.setHiddenApiPolicy = async function setHiddenApiPolicy (value) {
-  await this.setSetting('global', 'hidden_api_policy_pre_p_apps', value);
-  await this.setSetting('global', 'hidden_api_policy_p_apps', value);
-  await this.setSetting('global', 'hidden_api_policy', value);
+  const cmd = HIDDEN_API_POLICY_KEYS.reduce((acc, key) => {
```

Comment:
I'd rather reduce the command to an array and then join its items with `;`. This way we wouldn't have an extra semicolon at the end